WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -532,4 +488,56 @@ private static int GetInputTarget(int utxoCount, WasabiRandom rnd)
 
 		return targetInputCount;
 	}
+
+	private async Task<IEnumerable<TxOut>> ProgressOutputRegistrationPhaseAsync(uint256 roundId, ImmutableArray<AliceClient> registeredAliceClients, CancellationToken abortToken)
```

Comment:
I think the `CancellationToken` in the methods sigatures should be called `cancellationToken` everywhere. 

The cancellation token corresponding to the internal linked cancellation source could be called different like `linkedCts` or some other ugly name.

Reply:
ACK